src/Controller/WhoWeWorkWithController.php

```diff
@@ -108,6 +108,12 @@ public function whoWeWorkWithAction(Request $request) : Response
                 'type' => 'image/svg+xml',
                 'uri' => 'https://glencoesoftware.com/',
             ],
+            [
+                'name' => 'Hypothesis',
+                'filename' => 'hypothesisLogo',
+                'type' => 'image/svg+xml',
+                'uri' => 'https://hypothes.is',
```

Comment:
For consistency add a trailing slash, and rename the file to just `hypothesis`.

Reply:
Completed